avocado/utils/distro.py

```diff
@@ -220,8 +253,11 @@ def release(self):
                 release = match.groups()[1]
         return release
 
-    def get_distro(self):
+    def get_distro(self, session=None):
```

Comment:
IIUC, this object has already a `self.session` that is set during `__init__()`. What is the difference between the first and this?